Rack Middleware for Tracking Page Views

satoryu opened the pull request:
This pull-request provides a tiny rack middleware which inserts a script code to a content for tracking page views with application insights JavaScript SDK.  
This is inspired from this document:
 * [Azure Application Insights for JavaScript web apps | Microsoft Docs](https://docs.microsoft.com/en-us/azure/azure-monitor/app/javascript?toc=/azure/azure-monitor/toc.json#add-the-sdk-script-to-your-app-or-web-pages)

Commit:
Merge pull request #55 from Microsoft/develop

Release 0.5.6

satoryu committed:
Embeds javascript into the head to track page view with app insights

satoryu committed:
Add test code for the new rack middleware

SergeyKanzhelev commented on lib/application_insights/rack/track_page_view.rb:
```diff
@@ -0,0 +1,46 @@
+
+module ApplicationInsights
+  module Rack
+    class TrackPageView
```
perhaps it can be named differently. Typically `TrackFoo` method results in sending item. Maybe "InjectJavaScriptTracking"?

satoryu replied:
Thanks. As you pointed out, it was wrong name. so I renamed it you suggested to me.

SergeyKanzhelev reviewed (changes requested):
I suggested another name. What do you think?

satoryu committed:
Rename the middleware.

Refer to the following comment:
https://github.com/Microsoft/ApplicationInsights-Ruby/pull/64#discussion_r244918319

SergeyKanzhelev commented on lib/application_insights.rb:
```diff
@@ -5,5 +5,6 @@
 module ApplicationInsights
   module Rack
     autoload :TrackRequest, "application_insights/rack/track_request"
+    autoload :TrackPageView, "application_insights/rack/track_page_view"
```
I realize after merge that this wasn't renamed. Can you send separate PR please?

satoryu replied:
oh, sorry. I'll send new PR later.
